sda-commons-client-jersey/README.md

```diff
@@ -213,6 +213,104 @@ In addition, each client can consume the standard [proxy system properties](http
 Please note that a specific proxy configuration in the `HttpClientConfiguration` disables the proxy system properties for the client using that configuration.
 This can be helpful when all clients in an Application should use the same proxy configuration (this includes the clients that are used by the [`sda-commons-server-auth` bundle](../sda-commons-server-auth).
 
+## OIDC Client
+
+This module also provides support for requesting OIDC access tokens from
+respective providers (e.g. Keycloak) for subsequent requests to other services.
+
+### Usage
+
+The [`OidcClient`](./src/main/java/org/sdase/commons/client/jersey/oidc/OidcClient.java) can be instantiated
+with a Jersey [`ClientFactory`](../sda-commons-client-jersey/src/main/java/org/sdase/commons/client/jersey/ClientFactory.java)
+and the respectable [`OidcConfiguration`](./src/main/java/org/sdase/commons/client/jersey/oidc/OidcConfiguration.java).
+The client also implements caching that is configured according to the access tokens expiration time claim.
+
+```java
+OidcClient oidcClient = new OidcClient(jerseyClientBundle.getClientFactory(), configuration.getOidc());
+OidcResult oidcResult = oidcClient.createAccessToken();
+String accessToken = oidcResult.getAccessToken();
+```
+
+An easy way to integrate the OidcClient into existing PlatformClients is by creating a ClientRequestFilter
+and passing it to the PlatformClientBuilder:
+
+
+```java
+public class OidcRequestFilter implements ClientRequestFilter {
```

Comment:
Shouldn't we provide this class instead of just given the example code in the documentation?

Reply:
Im not sure, I'd rather just give an example and leave it up to the developer how he wants to implement it into the application.

Reply:
Maybe it is something for a follow up PR when it is needed/used in more than one service?